presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -1312,29 +1312,6 @@ public void testLambdaInSubqueryContext()
         analyze("SELECT apply(x, z -> y * x) FROM (SELECT 10 x, 3 y)");
     }
 
-    @Test
-    public void testLambdaWithAggregationAndGrouping()
```

Comment:
Why do we need to move this to ATQ? They should be fine staying in this file.

Reply:
Lambda is not analyzed at analysis time. The types are resolved in SignatureBinder later. Tests are moved to `AbstractTestQueries`.

Reply:
Moved back